app/src/main/res/values-de/strings.xml

```diff
@@ -204,50 +203,50 @@
 	<string name="game_level_medium">Medium</string>
 	<string name="game_level_hard">Schwer</string>
 
-    <string name="theme_default">Default</string>
-    <string name="theme_paper">Paper</string>
-    <string name="theme_graph_paper">Graph Paper</string>
-    <string name="theme_light">Light</string>
-    <string name="theme_paper_light">Paper Light</string>
-    <string name="theme_graph_paper_light">Graph Paper Light</string>
-    <string name="theme_high_contrast">High Contrast</string>
-    <string name="theme_inverted_high_contrast">Inverted High Contrast</string>
-    <string name="theme_custom">Custom Theme</string>
+    <string name="theme_default">Standard</string>
+    <string name="theme_paper">Papier</string>
+    <string name="theme_graph_paper">Karopapier</string>
+    <string name="theme_light">Hell</string>
+    <string name="theme_paper_light">Papier Hell</string>
+    <string name="theme_graph_paper_light">Karopapier Hell</string>
+    <string name="theme_high_contrast">Hoher Kontrast</string>
+    <string name="theme_inverted_high_contrast">Invertierter Hoher Kontrast</string>
+    <string name="theme_custom">Eigenes Thema</string>
 
-    <string name="screen_custom_theme">Custom Theme colors</string>
-    <string name="screen_custom_theme_summary">Customize colors of the Custom Theme.</string>
-    <string name="screen_custom_theme_summary_disabled">Choose \"Custom Theme\" in \"Theme\" setting above to enable this section.</string>
+    <string name="screen_custom_theme">Eigene Themenfarben</string>
+    <string name="screen_custom_theme_summary">Verändere Farben des eigenen Themas.</string>
+    <string name="screen_custom_theme_summary_disabled">Wählen Sie \"Eigenes Thema\" in der Themenauswahl, um diese Einstellungen zu Aktivieren.</string>
 
-    <string name="default_lineColor">Line</string>
-    <string name="default_lineColor_summary">Border color around a cell.</string>
+    <string name="default_lineColor">Rahmen</string>
```

Comment:
i would prefer "Linie" hier instead of "Rahmen". Could become confusing when reading the variable